include/dxc/Support/dxcapi.extval.h

```diff
@@ -0,0 +1,20 @@
+#include "dxc/Support/dxcapi.use.h"
+#include "dxc/dxcapi.h"
+
+class DxcDllExtValidationSupport : public dxc::DxcDllSupport {
+  // DxcDllExtValidationSupport manages the
+  // lifetime of dxcompiler.dll, while the member, m_DxilSupport,
+  // manages the lifetime of dxil.dll
+  dxc::DxcDllSupport m_DxilSupport;
```

Comment:
Let's be consistent with naming.

```suggestion
  dxc::DxcDllSupport DxilSupport;
```